app_flutter/test/utils/output.dart

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) 2019 The Chromium Authors. All rights reserved.
```

Comment:
nit: 2020

Reply:
I started this file by copying and pasting another so it's a derivative work of another file. I copied that file's copyright date.